modules/cnv-vm-rdp-console-web.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * TBD
+
+[[cnv-vm-rdp-console-web]]
+= Connecting to the RDP console
+
+The desktop viewer console, which utilizes the Remote Desktop Protocol (RDP), provides a better console experience for connecting to Windows virtual machines. A layer 2 network interface must be attached to the Windows virtual machine, and the QEMU guest agent, which is part of the VirtIO drivers, must be installed on the virtual machine to report the IP address of the network interface.
+
+Connect to the desktop viewer console of a running Windows virtual machine from the *Consoles* tab in the *Virtual Machine Details* screen of the web console. 
+
+.Prerequisites
+
+* A running Windows virtual machine with the QEMU guest agent installed. The `qemu-guest-agent` is included in the VirtIO drivers.
+* A layer 2 vNIC attached to the virtual machine. 
+* An RDP client installed on a machine on the same network as the Windows virtual machine. 
+
+.Procedure
+
+. Click *Workloads* -> *Virtual Machines* from the side menu.
+. Select a Windows virtual machine.
+. Click *Consoles*. 
+. Click the *Consoles* drop-down list and select *Desktop Viewer*.
```

Comment:
. From the \*Consoles\* list, select *Desktop Viewer*.

> Label: Refer to a drop-down list by its label, followed by _list_.

Remove all instances of "drop-down list."